iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -56,10 +57,15 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
 		testEditorRoleCrn string = "crn:v1:bluemix:public:iam::::role:Editor"
 		testServiceName   string = "iam-groups"
 
-		testCustomRoleId   string = ""
-		testCustomRoleETag string = ""
-		testCustomRoleName string = "TestGoRole" + strconv.Itoa(rand.Intn(100000))
-		testServiceRoleCrn string = "crn:v1:bluemix:public:iam-identity::::serviceRole:ServiceIdCreator"
+		testCustomRoleId          string = ""
+		testCustomRoleETag        string = ""
+		testCustomRoleName        string = "TestGoRole" + strconv.Itoa(rand.Intn(100000))
+		testServiceRoleCrn        string = "crn:v1:bluemix:public:iam-identity::::serviceRole:ServiceIdCreator"
+		testPolicyTemplateID      string = ""
+		testPolicyTemplateETag    string = ""
+		testPolicyTemplateVersion string = ""
+		// change testPolicyAssignmentId id after prod account setup
+		testPolicyAssignmentId = "442c6fc4-2f74-41b2-bf4d-98342614cd22"
```

Comment:
Why not use a flow like:
1. list policy assignments 2. Store an assignment id, 3. use stored id to call get assignment by id? 
I think it is okay to hardcode an ID here, but I'm not sure if it matters whether it is production or test environment. These tests are for us to run locally and we can give production credentials in the `iam_policy_management.env` when we test locally to verify works in production.

Reply:
I believe it is better to use assignment id from `list policy assignments`, as same `assignment id`  will not be available in different accounts so even if we hardcode it will restricted to use same account.

Thanks i'll update

Reply:
I updated the tests

Reply:
I think your reasoning sounds good. We can try to make sure we create an assignment in the account we use for testing, but luckily there are some we can refer to currently. 
Thanks for making that change.

Reply:
I might not understand 100% of this discussion, but it sounds like you need to use an id of a pre-existing account that your tests rely on.  The int tests and examples for this service already retrieve the "TEST_ACCOUNT_ID" config property from the .env file during startup. It is stored in the `testAccountID` variable in the integration tests and in `exampleAccountID` in the examples code.
It's way better to retrieve this value from the .env file than to keep having to update the hard-coded id within the actual source code.